dev/artifacts/Products/Products.recipes

```diff
@@ -7,8 +7,82 @@
 
 import 'Product.schema'
 import 'ShowProducts.manifest'
+import 'Recommend.manifest'
```

Comment:
Do we need all the changes in here? This was a straight copy over from what we had in sjmiles/arc-stories.

Reply:
we aren't using sjmiles/arc-stories for gift shopping, we were using 
https://github.com/PolymerLabs/arcs-cdn/blob/gh-pages/dev/apps/web/arc.manifest#L16
https://github.com/mmandlis/arcs-stories/blob/master/0.3/Products/Products.recipes

Reply:
Ahhh. Great!